nubis/puppet/files/proxy/confd-squid3/conf.d/squid_allow_domains.toml

```diff
@@ -0,0 +1,10 @@
+[template]
+src = "squid_allow_domains.tmpl"
+dest = "/etc/squid3/allow_domains.txt"
+prefix = "/%%STACK%%/%%ENVIRONMENT%%"
```

Comment:
Now that this is deployed as a sub-stack of the main stack, %%STACK%% in confd will end up being the name of the sub-stack, something really ugly (i.e us-west-2-vpc-StageVPCStack-16CF72VSC8SHF-NATStack-VYX4EPU6OWRL/) and not discoverable. I'd go with %%PROJECT%% instead (i.e. nubis-nat-stage)